src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const i of actions) {
+    for (const key in i) {
```

Comment:
Use correct naming. Second loop is redudant. Use switch/case for `action.type`
```suggestion
  for (const action of actions) {
```